src/settings-ui/Settings.UI/Views/AlwaysOnTopPage.xaml

```diff
@@ -1,141 +1,148 @@
-                                            <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Custom_Color"/>
-                                            <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Windows_Default"/>
-                                        </ComboBox>
-                                    </controls:Setting.ActionContent>
-                                </controls:Setting>
+                <controls:SettingsGroup
+                    x:Uid="AlwaysOnTop_Behavior_GroupSettings"
+                    IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
+                    <labs:SettingsExpander
+                        x:Uid="AlwaysOnTop_FrameEnabled"
+                        HeaderIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xEB3C;}"
+                        IsExpanded="True">
+                        <ToggleSwitch
+                            x:Uid="ToggleSwitch"
+                            IsOn="{x:Bind Mode=TwoWay, Path=ViewModel.FrameEnabled}" />
+                        <labs:SettingsExpander.Items>
+                            <labs:SettingsCard
+                                x:Uid="AlwaysOnTop_FrameColor_Mode"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}">
+                                <ComboBox
+                                    MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                    SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource BoolToComboBoxIndexConverter}}">
+                                    <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Custom_Color" />
+                                    <ComboBoxItem x:Uid="AlwaysOnTop_Radio_Windows_Default" />
+                                </ComboBox>
+                            </labs:SettingsCard>
+                            <labs:SettingsCard
+                                x:Uid="AlwaysOnTop_FrameColor"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource ReverseBoolToVisibilityConverter}}">
+                                <controls:ColorPickerButton SelectedColor="{x:Bind Path=ViewModel.FrameColor, Mode=TwoWay}" />
+                            </labs:SettingsCard>
+                            <labs:SettingsCard
+                                x:Uid="AlwaysOnTop_FrameThickness"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.FrameEnabled}"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.FrameAccentColor, Converter={StaticResource ReverseBoolToVisibilityConverter}}">
```

Comment:
I don't think the Visibility of the Thikness-Slider is depending on the {Windows/Custom} accent color, am I right?

Reply:
Thanks, fixed!